mne/decoding/tests/test_time_gen.py

```diff
@@ -241,6 +241,10 @@ def test_generalization_across_time():
 
     # TODO JRK: test GAT with non-exhaustive CV (eg. train on 80%, test on 10%)
 
+    # Make CV with some empty tests to ensure that the folds are passed
+    gat.cv_.test_folds[gat.cv_.test_folds == 1] = 0
+    gat.predict(epochs)
```

Comment:
I don't know much about this, but I agree it's important to have empty cases handled gracefully -- it often comes up during parallelization if you have more jobs/cores than slices of data to operate on. This sounds slightly different, but the idea I think is the same, that it shouldn't bomb. It does sound to me like having empty folds is probably not supposed to happen in this case, though -- if that's true and it's likely indicative of some user error, +1 for warning (or error if it should really never happen) from me.